steamsale.py

```diff
@@ -2,26 +2,24 @@
 
 """ A script that parses your Steam wishlist and finds discounts """
 
+from __future__ import print_function
+from __future__ import unicode_literals
 import sys
 import requests
-import yaml
-import os.path
 from re import sub, search
 from getopt import getopt, GetoptError
-from BeautifulSoup import BeautifulSoup
+from bs4 import BeautifulSoup
 from termcolor import colored
 
-
 class Wishlist(object):
     """ Class representing a Steam wishlist """
     def __init__(self, steam_id):
         if steam_id.isdigit():
-            url = 'http://steamcommunity.com/profiles/%s/wishlist' % steam_id
+            url = 'http://steamcommunity.com/profiles/{}/wishlist'.format(steam_id)
         else:
-            url = 'http://steamcommunity.com/id/%s/wishlist' % steam_id
+            url = 'http://steamcommunity.com/id/{}/wishlist'.format(steam_id)
         req = requests.get(url)
-        self.soup = BeautifulSoup(req.content,
-                convertEntities=BeautifulSoup.HTML_ENTITIES)
+        self.soup = BeautifulSoup(req.content, 'lxml')
```

Comment:
Could you also change `lxml` to `html.parser`? Otherwise lxml needs to be added to requirements, but it seems to work fine with `html.parser`.